### ISSUE TITLE
[Accessibility][Usable]: All emails together dialog does not close with ESC in scan mode of narrator

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10539]
Breakout from #8061

Tracking all of the issues brought up with the 2/20 accessibility pass on Dialog control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8704792b-ff92-40ce-969f-93c02c8b96a9%7D&action=default&uid=%7B8704792B-FF92-40CE-969F-93C02C8B96A9%7D&ListItemId=1824&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

**Comment by jdhuntington** [comment]
Dialog now closes with `escape`, even when in scan mode. Unable to find specific fix, but unable to repro.
